firmware-nucleo/smart-air-hockey-table/App/driver_led.c

```diff
@@ -52,20 +51,14 @@ void Driver_LED_SetColor(uint_fast8_t col, uint_fast8_t row, Color color)
 	// Transform to a per-channel linear index
 	pixel_index %= LED_MATRIX_PIXEL_COUNT / LED_CHANNELS;
 
-	// Offset of each color channel within the 24 buffer values that define a pixel
-	uint_fast32_t g_offset = pixel_index * LED_COLOR_DATA_SIZE;
-	uint_fast32_t r_offset = g_offset + 8;
-	uint_fast32_t b_offset = g_offset + 16;
+	// Color channel base address with pointer magic
+	uint64_t *color_base = (uint64_t *) &(led_buffer[channel][pixel_index * LED_COLOR_DATA_SIZE]);

```

Comment:
?

Reply:
test it Wednesday and merge it day of :)